doc/iterations/2025.4/add_loading_indicator_for_downloads.adoc

```diff
@@ -0,0 +1,41 @@
+= (S-M) Add loading indicator for downloads
+
+== Problem
+
+Projects can be downloaded through the contextual menu of the `/projects` page and of the `/project/$projectId/edit` page.
+Documents can be downloaded through the contextual menu in the Explorer view.
+
+These operations can take a bit of time as the backend produces the corresponding file (zip file for projects, serialized file for the document, etc.), especially on larger projects and documents.
+
+During these operations, the user has no feedback that something is actually going on, resulting in the possibility of them getting confused (has an issue occurred?) and/or leaving the page or re-trying the operation.
```

Comment:
Technically since it's a regular HTTP download, they have the feedback from their web browsers